src/app/pages/components/components.module.ts

```diff
@@ -2,11 +2,12 @@ import { NgModule } from '@angular/core';
 
 import { TreeModule } from 'angular-tree-component';
 import { ToasterModule } from 'angular2-toaster';
-
 import { ThemeModule } from '../../@theme/theme.module';
 import { ComponentsRoutingModule, routedComponents } from './components-routing.module';
+import { InputComponent } from './tree/input.component';
```

Comment:
let's rename it. Input is too generic, could you come up with something a bit more specific to its usage?